packages/ord-connect/src/SampleApp.tsx

```diff
@@ -14,14 +15,15 @@ import "./style.css";
 
 function TestControls() {
   const { send, error: sendError, loading: isSending } = useSend();
+  const { relay, error: relayError, loading: isRelaying } = useRelay();
```

Comment:
```suggestion
  const { createTransaction, error: createTransactionError, loading: isCreatingTransaction } = useCreateTransaction();
  const { relay, error: relayError, loading: isRelaying } = useRelay();
```

i think these 2 hooks are confusing now, so suggesting the change above.

The definition would be that a transaction is something we can create an input and output with, then we relay the transaction.

or alternatively, `useTransaction` which contains `createTransaction` and `relay` but this breaks the pattern of error and loading

Reply:
i think making the hook to be so generic (accept input and output) would not be our use case because we need the checking on the server. For now i think we can rename it to `useCreateSendTransaction`? because essentially it will only accept the receiver address and amount to send

Reply:
my bad, i mean we only need to rename that hook - i think `useCreateTransaction` would be good (it has a fixed input)

It creates a transaction which accepts the receiver address and amount to be sent, but doesn't send the transaction

Reply:
I think instead of exposing relay as its own hook, we can just add a param in `send` function from `useSend` to accept like `relay:boolean`? if relay is `false` we will create the PSBT for sending an amount without relaying it.

Reply:
updated to remove new relay hook and added a new param